charts/cluster-addons/templates/monitoring/loki-stack.yaml

```diff
@@ -1,86 +1,101 @@
-{{- if and .Values.monitoring.enabled .Values.monitoring.lokiStack.enabled }}
+{{- if and .Values.monitoring.enabled .Values.monitoring.loki.enabled }}
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "cluster-addons.componentName" (list . "loki-stack") }}-config
+  name: {{ include "cluster-addons.componentName" (list . "loki") }}-config
   labels:
-    {{- include "cluster-addons.componentLabels" (list . "loki-stack") | nindent 4 }}
+    {{- include "cluster-addons.componentLabels" (list . "loki") | nindent 4 }}
     addons.stackhpc.com/watch: ""
 stringData:
   defaults: |
     loki:
       serviceMonitor:
         enabled: true
-    promtail:
+      monitoring:
+        dashboards:
+          enabled: false
+
+  overrides: |
+    {{- toYaml .Values.monitoring.loki.release.values | nindent 4 }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "cluster-addons.componentName" (list . "promtail") }}-config
+  labels:
+    {{- include "cluster-addons.componentLabels" (list . "promtail") | nindent 4 }}
+    addons.stackhpc.com/watch: ""
+stringData:
+  defaults: |
+    config:
       serviceMonitor:
         enabled: true
-      # Get promtail to scrape systemd services
```

Comment:
We seem to have lost this explanatory comment in the new config below, can we re-add it please?